src/Angor/Shared/ProtocolNew/Scripts/ProjectScriptsBuilder.cs

```diff
@@ -24,10 +24,11 @@ public Script BuildInvestorInfoScript(string investorKey)
             Op.GetPushOp(new PubKey(investorKey).ToBytes()));
     }
 
-    public Script BuildFounderInfoScript(string founderKey, string nostrPuKey)
+    public Script BuildFounderInfoScript(string founderKey, short keyType , string nostrPuKey)
     {
         return new Script(OpcodeType.OP_RETURN,
             Op.GetPushOp(new PubKey(founderKey).ToBytes()),
+            Op.GetPushOp(keyType),
```

Comment:
I see what you did here, I would expect key type to be the first param though.

This raises the question should we use multisig to represent both pubkey and eventid instead of opreturn.

Reply:
Yes I think we should, but we need to have it as a different task and plan it out when implementing silent payments